Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -35,7 +35,8 @@ WK_APPLEJPEGXL_PREPROCESSOR_DEFINITIONS_YES = USE_JPEGXL=1;
 
 // FIXME: remove the interop setting here when rdar://124492036 is resolved.
 WK_HAVE_SWIFT_CPP_INTEROP = HAVE_SWIFT_CPP_INTEROP=0;
-WK_HAVE_SWIFT_CPP_INTEROP[sdk=macos14*] = ;
+WK_HAVE_SWIFT_CPP_INTEROP[sdk=macosx14*] = ;
+WK_HAVE_SWIFT_CPP_INTEROP[sdk=macoscatalyst14*] = ;
```

Comment:
`macoscatalyst` is not a thing, Catalyst uses the same `macosx` SDK.

Reply:
aah.. fixed..